src/ops/inc_multihead_self_attention.cu

```diff
@@ -936,6 +936,9 @@ void compute_attention_kernel_prompt(IncMultiHeadSelfAttentionMeta const *m,
   for (int i = 0; i < bc->max_requests_per_batch(); i++) {
     if (bc->request_completed[i]) {
       continue;
+    } else if (tokens_previous_requests <= bc->num_generation_tokens) {
```

Comment:
shouldn't this line be `<` instead of `<=`?

Reply:
Yes，sorry for the mistake